tokenserver/views.py

```diff
@@ -101,15 +101,29 @@ def valid_assertion(request):
     request.metrics['token.assertion.verify_success'] = 1
     request.validated['assertion'] = assertion
 
-    # log the email with the request
-    email = request.validated['assertion']['email']
+    id_key = request.registry.settings.get("fxa.metrics_uid_secret_key")
+    if id_key is None:
+        id_key = 'insecure'
+    email = assertion['email']
+    fxa_uid_full = fxa_metrics_hash(email, id_key)
+    # "legacy" key used by heka active_counts.lua
+    request.metrics['uid'] = fxa_uid_full
     request.metrics['email'] = email
 
-    # Include a unique FxA identifier in the logs, but obfuscate
-    # it for privacy purposes.
-    id_key = request.registry.settings.get("fxa.metrics_uid_secret_key")
-    if id_key:
-        request.metrics['uid'] = fxa_metrics_uid(email, id_key)
+    # "new" keys use shorter values
```

Comment:
Is this to keep the size of the token down?  How big is the token now with these extra fields in it?